modules/ingester/ingester_test.go

```diff
@@ -211,6 +211,42 @@ func TestWal(t *testing.T) {
 	}
 }
 
+//func TestSearchWAL(t *testing.T) {
```

Comment:
Do we want to delete this commented out test?

Reply:
Hmm, I had to make a copy of `defaultIngester()` with some search data but lost track of it